js/admin.js

```diff
@@ -62,6 +64,23 @@ jQuery(document).ready(function($){
 		$(data).children().attr('typeField', 1);
 		$(this).parent().parent().remove();
 	});
+
+	const editListViewPage = 'admin_page_onoffice-editlistview';
+	if (onOffice.settings.page === editListViewPage) {
+		$('#send_form').on('click', updateRequiredAttributeZipActiveField);
+	}
+	function updateRequiredAttributeZipActiveField() {
+		if ($('#geofields').is(':visible')) {
+			const isCityActive = $('input[name=oopluginlistviews-cityactive]:checked').is(':visible');
+			const isZipActive = $('input[name=oopluginlistviews-zipactive]:checked').is(':visible');
+			$('input[name=oopluginlistviews-zipactive]').attr('required', 'required');
+
+			if (isCityActive || isZipActive) {
+				$('input[name=oopluginlistviews-zipactive]').removeAttr('required');
+			}
+		}
+	}
+
```

Comment:
Please revert these changes.
More info in https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/641#issuecomment-1945901495